app/assets/javascripts/admin/admin_rest_api.js

```diff
@@ -427,7 +427,7 @@ export function editAnnotation(
   annotationId: string,
   annotationType: APITracingType,
   data: $Shape<EditableAnnotationType>,
-): Promise<void> {
+): Promise<any> {
```

Comment:
The any can probably be replaced by a more specific type or does this pose problems?